### PR TITLE
restrict the html file

### PR DESCRIPTION
**Pull request by lujiefsi**

### CODE REVIEW

**Comment by Zeng1998** [comment]
hello, why the html file is not allowed?

**Comment by lujiefsi** [comment]
> hello, why the html file is not allowed?

HI:

we send the security issuse to stevenlgtm@gmail.com

File: server/resource.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"net/url"
 	"strconv"
+        "strings"
```
Comment (jxpsert) [inline server/resource.go:10]: This line is prefixed by a few spaces, instead of a tab. Replace it and the check should succeed!

**Comment by lujiefsi** [comment]
Is there any way to see what errors happen?

**Review by boojack — APPROVED** [review]
LGTM for now. But I think we could return raw text for html resource instead of disabling it.

**Comment by lujiefsi** [comment]
could you verify the report on https://huntr.dev/  ?

**Comment by jxpsert** [comment]
Where? Do you have a direct link?

**Pull request merged**